### DISCUSSION TITLE
ilogtail-1.8.0 容器内编译插件出错

### DISCUSSION THREAD

**Discussion by ZZZZZZTong** [alibaba/ilogtail#1206] [Help] [answered]
参考知乎用户手册进行容器内插件编译时报错哇。https://zhuanlan.zhihu.com/p/552057171
执行make plugin_local 时在plugin_build.sh 第82行sudo mv libzstd__.a ${lib_name}
出现 mv: cannot stat ‘libzstd__.a’: No such file or directory
执行make vendor 并未报错
各位大佬知道这是什么原因吗？
错误前几行输出：
CC obj/conf_6d55ba4fc88ee27f3b1ebc21240b9938/static/zdict.o
compiling single-threaded static library 1.5.2
make[2]: Leaving directory `/root/go/pkg/mod/github.com/valyala/gozstd@v1.17.0/zstd/lib'
mv zstd/lib/libzstd.a libzstd_linux_amd64.a
make[1]: Leaving directory `/root/go/pkg/mod/github.com/valyala/gozstd@v1.17.0'
**mv: cannot stat ‘libzstd__.a’: No such file or directory**

**Comment by liangry** [comment, marked as answer]
没有遇到类似的错误，不过链接中的镜像可能有点旧，用最新版本镜像试试：sls-opensource-registry.cn-shanghai.cr.aliyuncs.com/ilogtail-community-edition/ilogtail-build-linux:gcc_9.3.1-1

**Reply by ZZZZZZTong** [reply]
(๑•̀ㅂ•́)و✧  谢谢大佬
